### ISSUE TITLE
Pinch to zoom on Safari mac doesn't consider modals

### ISSUE THREAD

**Issue by h7y** [excalidraw/excalidraw#3311]
## Demo
![Demo](https://media.giphy.com/media/iLNPDuyoKRfmpLqoFI/giphy.gif)

**Comment by dwelle** [comment]
What do you think should be the appropriate behavior? IMO it shouldn't zoom the canvas, but we also don't want to zoom in on the dialog as it would then break the canvas.

**Comment by h7y** [comment]
On Chrome & Firefox, it zooms in on the dialog now. If I close the modal while it is zoomed in, pinching out to zoom out doesn't work till I open the dialog again.

I think we should fix it by allowing zooming in while a dialog is open but reset zoom on exiting it.

**Comment by dwelle** [comment]
> I think we should fix it by allowing zooming in while a dialog is open but reset zoom on exiting it.

Not really possible to manipulate the browser zoom level programmatically. There are some non-cross-browser hacks, but they still don't change the zoom level.
